docs/content/en/docs/packages/overview.md

```diff
@@ -12,7 +12,14 @@ weight: 1
 Components of EKS Anywhere curated packages consist of a controller, a CLI, and artifacts.
 
 ### Package controller
-The package controller will install, upgrade, configure and remove packages from the cluster. The package controller will watch the packages and packagebundle custom resources for the packages to run and their configuration values. The package controller only runs on the management cluster and manages packages on the management cluster and on the workload clusters.
+The package controller will install, upgrade, configure and remove packages from the cluster.
+
+The package controller performs these actions by watching the package and packagebundle custom resources to determine which packages to run and their appropriate configuration values.
+
+Packages CRD resources map to helm charts that the package controller uses to install packages workloads (such as cluster-autoscaler or metrics-server) on your clusters. The packagebundle object is the mapping between the package name and the specific helm chart and images that will be installed.
+
+The package controller only runs on the management cluster and manages packages on the management cluster and on the workload clusters.
+To learn more about how to install packages on both management and workload clusters, see below.
 
 Package release information is stored in a package bundle manifest. The package controller will continually monitor and download new package bundles. When a new package bundle is downloaded, it will show up as update available and users can use the CLI to activate the bundle to upgrade the installed packages.
```

Comment:
> The package controller will continually monitor and download new package bundles 

Can users configure this downloading behavior?

> When a new package bundle is downloaded, it will show up as update available 

Where is this "update available"? How can a user check for that?

> users can use the CLI to activate the bundle to upgrade the installed packages.

So packages won't automatically update unless the user explicitly performs this action?

Reply:
> Can users configure this downloading behavior?
no, it is not configurable.

> So packages won't automatically update unless the user explicitly performs this action?
correct

> Where is this "update available"? How can a user check for that?
would you be open to us filing a ticket to answer this question? it will require some investigation and I'm not sure I'll be able to get to it before I head out for PTO. I'd like to get the rest of this merged if possible.